src/modules/project-config.ts

```diff
@@ -35,7 +36,7 @@ export class ProjectConfig implements ProjectConfigOptions {
 
     // project-wide variable configuration
     variables: Map<string, any> = new Map<string, any>();
-    cliVersion: string | undefined;
+    cliVersion: string = 'v0.6.2';
```

Comment:
Is it possible to read this dynamically?

Reply:
i think it is only possible to use `this.config.version`in a Command class